sections/plate_notation.tex

```diff
@@ -77,43 +39,22 @@ \subsection{Author-Topic \gls{lda} and Category-Topic \gls{lda}}
-An example of the \gls{dag} structure used in this paper is in \autoref{fig:pachinko_dag} where each level of the \gls{pam} is shown.
-\vejleder[inline]{be more explicit if that is the same as original or different}
-In this paper, we use a layered \gls{pam} meaning that the \gls{dag} structure is divided into layers, with every node in each layer fully connected to every node in the next layer of the hierarchy.
-The first layer consists of only the root node, a topic which all documents are part of, and the bottom layer consists of leaf nodes, which are the only nodes to contain distributions over words, and they have no other topics in their distribution.
+In this paper, we construct a five-level \gls{pam}, following the format presented by \citet{li2006pachinko}.
+The \gls{dag} structure used in this paper is in \autoref{fig:pachinko_dag}, where each layer has been named.
+The plate notation for this model is shown in \autoref{fig:pachinko}
 
-We replicate some layers in our \gls{dag} structure based on the structure from the taxonomy field within our dataset, having some nodes represent a topic based on a specific taxonomy.
+Some layers in our \gls{dag} structure are duplicates of layers from the taxonomy field within our dataset.
+This allows us to have the some nodes represent a topic based on a specific taxonomy.
 To make the algorithm construct the topics to be based on our taxonomies, we introduce a novel locking mechanism for the Gibbs sampler which we use to run \gls{pam}.
 This mechanism is discussed further in \autoref{subsec:mod_pachinko}.
 
-We use a five-level pachinko tree structure, following the format presented by \citet{li2006pachinko}.
-The first layer is the root layer.
+The first layer of the \gls{dag} is the root layer.
 The last layer is the word layer consisting of one node for each word in the vocabulary of our corpus.
 The second and third layers will be constructed based on the entries of the first two positions in our taxonomy metadata, meaning there will be one node for each unique sub-taxonomy entry that is in the first or second position in the taxonomy sequence (e.g. "STEDER" and "Danmark", which is taken from "STEDER/Danmark/Aalborg", but not "Aalborg" since it is in the third position).
-We only use the first two layers for this, as these are among the most informative, and because introducing even more layers would slow down the training exponentially, since the probability of all possible combinations of topics needs to be sampled for every word during training. \vejleder[inline]{can you back this up with experiments or hard evidence?}
+We only use the first two layers for this, as these are among the most informative, and because introducing even more layers would slow down the training significantly, since the probability of all possible combinations of topics needs to be sampled for every word during training. \vejleder[inline]{can you back this up with experiments or hard evidence?}
```

Comment:
```suggestion
We only use the first two layers for this, as these are among the most informative, and because introducing even more layers would slow down the training significantly, since the probability of all possible combinations of topics needs to be sampled for every word during training. \vejleder[inline]{can you back this up with experiments or hard evidence?}
```
måske kom med 12 timer vs 130 timer I run time ?